Sources/DulocCastle/ASN1/Asn1Constants.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Asn1Constants.swift
+//  
+//
+//  Created by Kevin Miller on 7/23/22.
+//
+
+import Foundation
+
+struct Asn1Constants {
+    
+    static let OID_REGISTRY_FILE = "OidRegistryBuiltIn"
+    static let OID_REGISTRY_FILE_EXT = "txt"
```

Comment:
Consider switching to universal type identifiers.

Reply:
Let's just drop the file loading altogether